ioctl/newcmd/config/config_reset.go

```diff
@@ -5,3 +5,28 @@
 // License 2.0 that can be found in the LICENSE file.
 
 package config
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/iotexproject/iotex-core/ioctl"
+)
+
+// NewConfigReset resets the config to the default values
+func NewConfigReset(client ioctl.Client, defaultConfigFileName string) *cobra.Command {
+	return &cobra.Command{
+		Use:   "reset",
+		Short: "Reset config to default",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
+			info := newInfo(client.Config(), defaultConfigFileName)
+			err := info.reset()
```

Comment:
1. combine in one line: newInfo(client.Config(), defaultConfigFileName).reset()

2. add unit test TestReset in config_test.go